### PR TITLE
Update for HyperSpy v1.5

### PR DESCRIPTION
**Pull request by francisco-dlp**
See https://github.com/hyperspy/hyperspy/pull/2180

Requires HyperSpy v1.5

### CODE REVIEW

File: hyperspy_gui_traitsui/utils.py [hunk 1]
```diff
@@ -3,10 +3,6 @@
 from hyperspy.ui_registry import register_widget
```
Comment (ericpre) [inline hyperspy_gui_traitsui/utils.py]: This import needs to be remove, because `register_widget` does not exist anymore.

File: setup.py [hunk 2]
```diff
@@ -74,13 +74,8 @@
...
-
-    # List additional groups of dependencies here (e.g. development
-    # dependencies). You can install these using the following syntax,
-    # for example:
-    # $ pip install -e .[dev,test]
-    # extras_require={
-    #     'test': ['pytest'],
-    # },
-)
+    install_requires=['hyperspy>=1.5.dev', 'traitsui>=6.0'],
```
Comment (ericpre) [inline setup.py]: Same as in https://github.com/hyperspy/hyperspy_gui_ipywidgets/pull/19#discussion_r307686020?

**Comment by ericpre** [comment]
Once my comments are addressed, this can be merged.

**Comment by francisco-dlp** [comment]
Done, thanks!

**Pull request merged**